src/MockResponses/setup.json

```diff
@@ -24,32 +24,54 @@
         "Response": "12345"
     },
     {
-        "Path": "/GetSiteDoseAvailability",
+        "Path": "/qflow/GetSiteDoseAvailability",
```

Comment:
This path is not correct. We are going directly to QFlow not via APIM

Reply:
Qflow is weird and this endpoint uses the same back end endpoint as '/availability' but gives a different model that is required for slot count by hour so it has to be this otherwise it won't work